### ISSUE TITLE
No window related triggers are kicking in

### ISSUE THREAD

**Issue by thewavelength** [digitalcreations/MaxTo#575]
**Describe the bug**
I think since the regex feature has been introduced, detection of windows is buggy. This goes for window moved, opened, activated and so on (all window related triggers I have tested). I can't get MaxTo to recognize any program. Doesn't matter if I use process name, window title or class and doesn't matter whether regex is enabled or not. The recipe isn't executing.

**To Reproduce**
I tried it with...

- `process name` = `foobar2000` (each with and without regex)
- `process name` = `foobar2000.exe` (each with and without regex)
- `window title` = `* foobar2000`

Here is a non-working configuration:
```
    {
      "name": "Automation foobar2000",
      "hotkey": null,
      "triggers": [
        {
          "type": "window:moved",
          "Window": "\\foobar2000.exe\\//\\//"
        }
      ],
      "ingredients": [
        {
          "command": "window:fit",
          "parameters": {
            "windows": "\\foobar2000.exe\\\\"
          }
        },
        {
          "command": "window:center",
          "parameters": {
            "window": "\\foobar2000.exe\\\\"
          }
        }
      ]
    }
```

**Expected behavior**
Should be triggered when moving.

**System information:**
- Windows version: Windows 10 Pro 1909 18363.836
- MaxTo version: 2.1.0-beta2

**Additional context**
There is no entry in the log file that the trigger has been detected.

**Comment by vegardlarsen** [comment]
I've confirmed there is a bug in 2.1.0-beta.2 that will prevent this from working in the same cases as when we cannot show the preview window of where a window will end up. This will be fixed.

**Comment by vegardlarsen** [comment]
This should be fixed for 2.1.0-beta.4.
